### ISSUE TITLE
./jb_run.js doesn't work

### ISSUE THREAD

**Issue by keiranmraine** [GMOD/jbrowse#1003]
As described here: https://github.com/GMOD/jbrowse/#install-from-github-repo-development

* OSX 10.11.6 (El Capitan)
* node v9.6.1 (also on v8.3.0, slightly different line num)

I've tried both a shallow and full clone:

```
$ git clone --depth 1 https://github.com/GMOD/jbrowse.git -b master
Cloning into 'jbrowse'...
...

$ cd jbrowse
$ npm install
npm WARN deprecated filesaver.js@0.2.0: Unmaintained namespace. Please use the file-saver package: https://www.npmjs.com/package/file-saver
npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.4.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
...
added 100 packages in 40.157s

$ ./jb_setup.js
NOTE: Legacy scripts wig-to-json.pl and bam-to-json.pl have removed from setup. Their functionality has been superseded by add-bam-track.pl and add-bw-track.pl. If you require the old versions, please use JBrowse 1.12.3 or earlier.
Installing javascript dependencies ...done
Gathering system information ...done
Installing Perl prerequisites ... done.

Formatting Volvox example data ... done.
To see the volvox example data, browse to http://your.jbrowse.root/index.html?data=sample_data/json/volvox.

Formatting Yeast example data ... done.
To see the yeast example data, browse to http://your.jbrowse.root/index.html?data=sample_data/json/yeast.

$ ./jb_run.js
events.js:112
      throw er; // Unhandled 'error' event
      ^

Error: listen EACCES 0.0.0.0:80
    at Object._errnoException (util.js:1003:13)
    at _exceptionWithHostPort (util.js:1024:20)
    at Server.setupListenHandle [as _listen2] (net.js:1336:19)
    at listenInCluster (net.js:1394:12)
    at Server.listen (net.js:1482:7)
    at Function.listen (/Users/kr2/GitHub/cgpJBrowseToolkit/jbrowse/node_modules/express/lib/application.js:618:24)
    at Object.<anonymous> (/Users/kr2/GitHub/cgpJBrowseToolkit/jbrowse/jb_run.js:54:5)
    at Module._compile (module.js:662:30)
    at Object.Module._extensions..js (module.js:673:10)
    at Module.load (module.js:575:32)
```

Any ideas?

**Comment by enuggetry** [comment]
Does it work with sudo?

**Comment by keiranmraine** [comment]
Ah, I see the problem. Didn't spot the port that was being attempted, used to these tools defaulting to ports that don't require sudo.

Should the doc indicate sudo is required or the script detect un-privileged user and set a different port?

**Comment by erasche** [comment]
Here's a PR to switch to an unprivileged port. https://github.com/GMOD/jbrowse/pull/1005

**Comment by rbuels** [comment]
merged #1005, which makes jb_run.js use 8080 by default, which it probably should always have
